src/client/components/savePressetModalBlock/savePressetModalBlock.js

```diff
@@ -0,0 +1,30 @@
+import React, { Fragment } from 'react';
+import PropTypes from 'prop-types';
+import ModalSavePressetesWindow from './modalSavePressetesWindow';
+import BackgroundUnderModelWindow from './backgroundUnderModelWindow';
+
+const SavePressetModalBlock = (props) => {
+  const autoFocusModalSavePressetesWindow = (elementFromRef) => {
+    if (elementFromRef) {
+      elementFromRef.focus();
+    }
+  };
+  const { showHideModalBlock } = props;
```

Comment:
You can destruct it in function parameters

Reply:
done)